assets/base.css

```diff
@@ -2,52 +2,52 @@
 
 :root,
 .color-background-1 {
-  --color-foreground: var(--color-base-text);
-  --color-foreground-rgb: var(--color-base-text-rgb);
-  --color-background: var(--color-base-background-1);
-  --color-background-rbg: var(--color-base-background-1-rgb);
-  --payment-terms-background-color: var(--color-base-background-1);
+  --color-foreground: rgb(var(--color-base-text));
+  --color-foreground-rgb: var(--color-base-text);
```

Comment:
I find this a bit confusing—I'd expect `-rgb` to be the one with the `rgb` function attached. Thoughts on instead having a single color, e.g. `--color-foreground: var(--color-base-text)` for both so we can rely on a single custom property for all references?

Reply:
Then the one thing about this approach as I was thinking about it is that we need to add `rbg(...)` around the `var(--color-base-text)` variables in a few files which then adds more lines/characters overall. 

But I do find it clearer to have one single custom property rather than 2.

Reply:
@chrisberthe Updated. [See details](https://github.com/Shopify/dawn/pull/97#issuecomment-875043775)